x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/query/VectorScriptDocValues.java

```diff
@@ -46,7 +46,11 @@ public BytesRef get(int index) {
 
     @Override
     public int size() {
-        throw new UnsupportedOperationException("vector fields may only be used via vector functions in scripts");
```

Comment:
Small comment: now that you can call `size` on a vector field, perhaps we could update the error message above to say "accessing a vector field's value through 'get' or 'value' is not supported".